src/commands/createFunction/FunctionSubWizard.ts

```diff
@@ -94,9 +91,8 @@ export class FunctionSubWizard {
                 }
             }
 
-            if ((!template.isHttpTrigger && !template.isSqlBindingTemplate) && !canValidateAzureWebJobStorageOnDebug(context.language) && !await getAzureWebJobsStorage(context, context.projectPath)) {
-                promptSteps.push(new AzureWebJobsStoragePromptStep());
-                executeSteps.push(new AzureWebJobsStorageExecuteStep());
+            if (context.newDurableStorageType) {
```

Comment:
Wait, any idea why this was changed? I don't feel like this is equivalent.

Reply:
Because we removed the `skipForNow` pattern we no longer need to create these resources during `createFunction`.